### PR TITLE
Consolidate the Oregon AGI logic according to the model standards

### LINKED ISSUE #3567: Consolidate the Oregon AGI logic according to the model standards
We currently use or_income_after_subtractions and or_income_after_additions which can be consolidated into one or_agi file, additionally the labels in the variables should be adjusted to reflect Oregon as opposed to the abbreviation OR

### PR DESCRIPTION
**Pull request by Kevintowne**
Fixes #3567

### CODE REVIEW

File: policyengine_us/variables/gov/states/or/tax/income/or_agi.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/or/tax/income/or_agi.py:1]: delete oh_income_after_additions

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: add changelog

File: policyengine_us/variables/gov/states/or/tax/income/or_income_after_subtractions.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/or/tax/income/or_income_after_subtractions.py:1]: delete the income_after_additions file

**Review by MaxGhenis — COMMENTED** [review]
Have we adopted `or_additions` as a more conventional name than `or_income_additions`?

**Comment by PavelMakarchuk** [comment]
> Have we adopted `or_additions` as a more conventional name than `or_income_additions`?

We have adopted `x_additions`, however we should consolidate as in early state modeling we used `x_agi_additions` and `x_income_additions`

**Review by MaxGhenis — APPROVED** [review]
OK I'll merge this and save that consolidation for another pr

**Pull request merged**